sangwoo/db/migrations/20230514053054_comments.sql

```diff
@@ -0,0 +1,15 @@
+-- migrate:up
+create table comments (
+  id int not null auto_increment,
+  content varchar(3000) not null,
+  user_id int not null,
+  post_id int not null,
+  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+	updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
+	PRIMARY KEY(id),
+  FOREIGN KEY (user_id) REFERENCES users(id),
+  FOREIGN KEY (post_id) REFERENCES posts(id)
+)
```

Comment:
- 해당 부분도 들여쓰기가 살짝 어긋나 있는 것 같습니다. 다른 파일에서도 들여쓰기 tab-size를 동일하게 맞춰주면 좋을 것 같네요!
- 외래키 제약 조건을 `CONSTRAINT ... `로 작성하여 제약 조건의 이름을 설정하고 추후 관리도 할 수 있도록 수정해주세요!
- 컬럼에 대해 필드 (타입)를 지정하고 속성을 지정하는 방식도 통일성을 이루면 좋을 것 같습니다. 대문자 혹은 소문자로 통일해주세요~!
```suggestion
create table comments (
  id int not null auto_increment,
  content varchar(3000) not null,
  user_id int not null,
  post_id int not null,
  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
  updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
  PRIMARY KEY(id),
  CONSTRAINT comments_user_id_fkey FOREIGN KEY (user_id) REFERENCES users(id),
  CONSTRAINT comments_post_id_fkey FOREIGN KEY (post_id) REFERENCES posts(id)
)
```